about.html

```diff
@@ -0,0 +1,140 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="about.css">
+    <title>JSOCIETY Summit 2023- Conference Web Page</title>
+</head>
+<body>
+<!--HEADER-->
+    <div class="background">
+        <header class="header">
+            <div class="dark-nav">
+                <ul>
+                    <li><a href="https://www.facebook.com/"><img src="images/facebook.png" alt="fcebook icon"></a></li>
+                    <li><a href="https://twitter.com/home"><img src="images/twitter.png" alt="twitter icon"></a></li>
+                    <li><a href="/">English</a></li>
+                    <li><a href="/">My Page</a></li>
+                    <li><a href="/">Log out</a></li>
```

Comment:
- [ ] Please note that the logout & community links are equidistant from the right edge. Kindly implement this change. 😺 
![des1](https://github.com/peter-mwovi/Capstone-project/assets/99656687/8d38d09b-acbb-40b3-a019-8a2472e5d1e9)